central/graphql/resolvers/image_vulnerabilities_test.go

```diff
@@ -270,34 +271,33 @@ func (s *GraphQLImageVulnerabilityTestSuite) TestImageVulnerabilitiesNonFixable(
 func (s *GraphQLImageVulnerabilityTestSuite) TestImageVulnerabilitiesFixedByVersion() {
 	ctx := SetAuthorizerOverride(s.ctx, allow.Anonymous())
 
-	vuln := s.getImageVulnerabilityResolver(ctx, "cve-2018-1#")
-
 	scopedCtx := scoped.Context(ctx, scoped.Scope{
```

Comment:
Please get rid of scopedCtx since it is no longer being used

Reply:
scopedCtx is needed to fetch the ImageVulnerabilityResolver that is component scoped by calling `getImageVulnerabilityResolver`.